app/services/SubmissionModelService.scala

```diff
@@ -43,15 +51,13 @@ class SubmissionModelService @Inject()(appConfig: AppConfig) {
         Left(FailedToConvertModel)
     }
 
-  private def buildMeta(messageSender: MessageSender, interchangeControlReference: InterchangeControlReference): Meta =
-    Meta(messageSender = messageSender, interchangeControlReference = interchangeControlReference)
-
   private def buildHeader(arrivalNotification: NormalNotificationMessage, procedureTypeFlag: ProcedureTypeFlag): Header =
```

Comment:
Should this be pulled out like Meta/buildMeta has?

Reply:
Not sure, I don't think it matters really, I don't want to create unnecessary changes that are more preferences than anything else